client/cypress/e2e/pages/flightupload.cy.js

```diff
@@ -136,12 +138,20 @@ describe("check flight upload page", () => {
     cy.get("Button").contains("Streckenmeldung absenden").should("be.disabled");
     cy.get("#acceptTermsCheckbox").check();
 
-    // Wait till all photos are uploaded
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(5000);
+    // Wait till all photos are uploaded (9)
+    for (let i = 1; i <= 9; i++) {
+      cy.wait("@post-photo");
```

Comment:
Wieso wird denn hier innerhalb einer Schleife gewartet?
Der Intercept differenziert doch nicht zwischen den einzelnen Calls. Das ist doch immer der gleiche oder nicht?

Reply:
Doch der sieht alle einzelnen. Kann man in der GUI auch gut sehen.